client/scss/components/_figure.scss

```diff
@@ -1,14 +1,18 @@
 //* @define figure
 
 .figure {
-  @include bleed(map-get($gutter-width, 'small'));
+  margin: 0;
 
-  @include respond-to('medium') {
-    @include bleed(map-get($gutter-width, 'medium'));
-  }
+  &:not(.figure--full) {
```

Comment:
To explain a little bit more clearly my concern here we are basically saying:
```JS
if (!full) { applyStyles(); }
```

Which allows everything through, so when we add new classes - this magically get's applied.

What can often be clearer is:
```JS
if (notFull /* or better, if 'slim' for example*/) { applyStyles(); }
```

So basically:
```SASS
.figure {
  margin: 0;
}
.figure--slim {
  /* All the other goodness */
}
```